integrationTests/cypress/support/step_definitions/dogu_integration_test_library_steps.js

```diff
@@ -0,0 +1,15 @@
+// Loads all steps from the dogu integration library into this project
+const doguTestLibrary = require('@cloudogu/dogu-integration-test-library')
+doguTestLibrary.registerSteps()
+
+When(/^the user clicks the dogu logout button$/, function () {
+    cy.get('#nx-header-signout-1144-btnEl').click();
+});
+
+Then(/^the user has administrator privileges in the dogu$/, function () {
+    cy.get('#button-1125-btnIconEl').should('be.visible')
```

Comment:
Just a really very small detail:
Here you use btnIconEl, in line 6 you only use btnEl. For a consistent implementation, perhaps either the button or the icon should be taken both times.